src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -473,6 +474,40 @@ public void notDistortContent() throws IOException {
     }
 
     /**
+     * RqMultipart.Base can produce parts with Content-Length.
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void producesPartsWithContentLength() throws IOException {
+        final RqMultipart.Base multipart = new RqMultipart.Base(
+            new RqFake(
+                Arrays.asList(
+                    "POST /h?a=4 HTTP/1.1",
+                    "Host: rtw.example.com",
+                    "Content-Type: multipart/form-data; boundary=AaB01x",
+                    "Content-Length: 100007"
+                ),
+                Joiner.on("\r\n").join(
+                    "--AaB01x",
+                    "Content-Disposition: form-data; name=\"t2\"",
+                    "",
+                    "447 N Wolfe Rd, Sunnyvale, CA 94085",
+                    "--AaB01x"
+                )
+            )
+        );
+        MatcherAssert.assertThat(
+            multipart.part("t2")
+                .iterator()
+                .next(),
+            new HmRqHeader(
+                "content-length",
```

Comment:
Just in case - HmRqHeader doesn't allow using right header name - "Content-Length". It expects it lower case.